src/cursor/cur_backup_incr.c

```diff
@@ -114,7 +118,15 @@ __curbackup_incr_next(WT_CURSOR *cursor)
           cb->granularity, WT_BACKUP_RANGE);
     } else if (btree == NULL || F_ISSET(cb, WT_CURBACKUP_FORCE_FULL)) {
         /* We don't have this object's incremental information, and it's a full file copy. */
-        WT_ERR(__wt_fs_size(session, cb->incr_file, &size));
+        WT_ERR(__wt_scr_alloc(session, 0, &buf));
```

Comment:
There's no reason to allocate a scratch buffer unless it's a log file?

Reply:
That is correct. That's the only one where we might have to prepend the log path.

Reply:
Sorry, I understand what you mean now...

Reply:
Good catch. I intended, in my mind, to only allocate if it's a log file. Fixed.